content/chat/setup.textile

```diff
@@ -0,0 +1,102 @@
+---
+title: SDK setup
+meta_description: "Install, authenticate and instantiate the Chat SDK."
+product: chat
+languages:
+  - javascript
+---
+
+Use these instructions to install, authenticate and instantiate the Chat SDK.
+
+h2(#authentication). Authentication
+
+An API key is required to authenticate with Ably. API keys are used either to authenticate directly with Ably using basic authentication, or to generate tokens for untrusted clients using "token authentication":/auth/token.
+
+<aside data-type='important'>
+<p>The examples use basic authentication to demonstrate features for convenience. In your own applications, basic authentication should never be used on the client-side as it exposes your Ably API key. Instead use token authentication.</p>
+</aside>
+
+"Sign up":https://ably.com/sign-up to Ably to create an API key in the dashboard or use the "Control API":/account/control-api to create an API key programmatically.
+
+API keys and tokens have a set of "capabilities":/auth/capabilities assigned to them that specify which operations, such as subscribe or publish can be performed on which resources. To use the Chat SDK, the API key requires the following capabilities depending on which features are being used:
+
+|_. Feature |_. Capabilities |
+| Send and receive messages | Publish, Subscribe |
+| Message history | Subscribe, History |
+| Online status | Subscribe, Presence |
+| Room occupancy | Subscribe, Channel Metadata |
+| Typing indicators | Publish, Subscribe, Presence |
+| Room reactions | Publish, Subscribe |
+
+When setting the capabilities for Chat, you need to apply them to either a wildcard resource, or a wildcard resource prefixed with the chat namespace, for example:
+
+* @[chat]*@ and @*@, or
+* @[*]*@
+
+h2(#install). Install
+
+The Chat SDK is built on top of the Ably Pub/Sub SDK and uses that to establish a connection with Ably.
+
+Both SDKs are available as NPM modules and via CDN.
+
+h3(#npm). NPM
+
+Install the Pub/Sub SDK and the Chat SDK:
+
+```[sh]
+npm install @ably/chat
+```
+
+Import the SDKs into your project:
+
+```[javascript]
+import * as Ably from 'ably';
+import ChatClient from '@ably/chat';
+```
+
+h3(#cdn). CDN
+
+Reference the Pub/Sub SDK and the Chat SDK within the @<head>@ of your HTML file:
```

Comment:
Do they have to be in `<head>`? Can they be async? To the minimum we should provide an integrity hash - I guess after CDN stuff is sorted?

Reply:
Yeah fair point. Need the final CDN path too.